### ISSUE TITLE
Dovecot tries to resolve the user domain with DNS

### ISSUE THREAD

**Issue by stephdl** [NethServer/dev#6808]
In a production system Dovecot sends out many DNS requests, trying to resolve the domain suffix corresponding to the LDAP user domain.

The request volume is high ~20000 per hour, it depends on the server workload.

Discussion
https://mattermost.nethesis.it/nethesis/pl/8k8jyq4s77y17bew4nkps178ee

**Comment by stephdl** [comment]
QA

- install 2 accounts provider, one LDAP and one SAMBA AD
- install mail : add-module ghcr.io/nethserver/mail:1.3.1-dev.2
- configure the mail server to use one account provider
- become mail1 : ssh mail1@::1
- verify the /etc/hosts of the dovecot container points to the IP of the domain controller

`podman exec -ti dovecot cat /etc/hosts`

```
 127.0.0.1	R4-pve.rocky9-pve4.org localhost localhost.localdomain localhost4 localhost4.localdomain4
::1	R4-pve.rocky9-pve4.org localhost localhost.localdomain localhost6 localhost6.localdomain6
127.0.0.1	cluster-leader
10.5.4.1	cluster-localnode
192.168.12.119	host.containers.internal

#user_domain
127.0.0.1 ad.rocky9-pve4.org
```

- change the account provider of the mail module : settings > general page
- verify that the account provider has changed inside the /etc/host s of the container

**Comment by nrauso** [comment]
tested: **VERIFIED**

**Comment by stephdl** [comment]
can we release @DavidePrincipi

**Comment by stephdl** [comment]
released as 1.3.1
